### PR TITLE
update milepost section on data page

### PR DESCRIPTION
**Pull request by gregbunce**
with newest udot mile reference post data

### CODE REVIEW

File: data/transportation/roads-system/index.html [hunk 1]
```diff
@@ -221,50 +216,31 @@ <h5>Related Resource</h5>
...
+    <p><code>FIELDS worth noting</code></p>
+    <ul>
+      <li><code>Legend</code> - Milepost number on sign
+      <li><code>Measure</code> - Location of the milepost along the LRS
+      <li><code>Station Label</code> - Highway name + direction of route along LRS (P = positive, milepost numbers increasing; N = negative, milepost numbers decreasing) + milepost number. For example the value 0015PM_271 translates to mile marker 217 on Interstate-15 on the north bound (P) side.
+    </ul>
+    <p>The Mile Reference Posts is an <a href="https://www.esri.com/en-us/arcgis/products/arcgis-roads-highways/overview">ESRI Roads and Highways Event</a> layer and supports the <a href="https://maps.udot.utah.gov/wadocuments/Apps/Stationing/">RP Stationing App</a>.</p>
+    <h5>Related Resource</h5>
+    <ul class="dotless">
+      <li><a href="https://data-uplan.opendata.arcgis.com/datasets/linear-measure-lm-tenth-opendata/explore">Linear-measure based mileposts</a> (derived from UDOT’s linear referencing system and contains tenth-mile points)
```
Comment (steveoh) [inline data/transportation/roads-system/index.html:239]: **suggestion**: should we add a link to the web api's milepost geocoding here?

**Review by steveoh — CHANGES_REQUESTED** [review]
Looks good. I might suggest giving the abridged version of the reference post vs linear reference as a convenience to the reader instead of linking out.

File: data/transportation/roads-system/index.html (hunk 1 above)
Comment (gregbunce) [reply data/transportation/roads-system/index.html:239]: good call.

Comment (gregbunce) [reply data/transportation/roads-system/index.html:239]: what's the best url
https://api.mapserv.utah.gov/  ?

Comment (steveoh) [reply data/transportation/roads-system/index.html:239]: https://api.mapserv.utah.gov/#geocoding

**Comment by gregbunce** [comment]
i'm happy with this pr.  let me know if i missed anything or if anyone wants changes.

i guess we'll wait to merge till the layer is open sgid.

**Comment by steveoh** [comment]
Let's wait for forklift and opensgid to sync everything and then we can merge this and the sgid index pr. Sound good?

**Comment by gregbunce** [comment]
Sounds great

**Review by steveoh — APPROVED** [review]

**Pull request merged**